.gitignore

```diff
@@ -23,3 +23,9 @@ server/manifest.go
 
 # notice
 .notice-work
+
+# Aider
+.aider*
+!.aider.conf.yml
```

Comment:
I don't think this should be excluded from the ignore, I have my anthropic api key in the conf file so I think it makes sense to ignore it. Also, aider configs are likely different for every user